### DISCUSSION TITLE
Seeding random numbers

### DISCUSSION THREAD

**Discussion by voroskoi** [aduros/wasm4#427] [Questions] [answered]
Hi,

I am having trouble with generating "real" pseudo numbers. My seed value is constant right now, which result the same layout on every restart (which is understandable).

People usually use time as the random number generator seed, but time is not available in wasm if I understand it correctly.

My best idea so far is having a startup screen which counts ticks before starting the game, so I can use that sort of random number as seed.

Is there a best practice for this?

Thanks!

**Comment by JerwuQu** [comment, marked as answer]
See #401
